pallets/delegation/src/lib.rs

```diff
@@ -323,25 +326,30 @@ impl<T: Trait> Module<T> {
 	pub fn is_delegating(
 		account: &T::AccountId,
 		delegation: &T::DelegationNodeId,
+		max_depth: u64,
 	) -> result::Result<bool, &'static str> {
+		if max_depth == 0 {
+			return Err("Reached max delegation search depth");
```

Comment:
1. This error is never checked in a test or am I missing something?
2. Do we want this style or rather open an else closure?

Reply:
1. I hid it well. 😄 https://github.com/KILTprotocol/mashnet-node/pull/109/files#diff-030cb87e9d69994cc367bbd6579734da355d6901ba5791f2c8997235a668c092R410-R411
2. In general I'm all for for doing one return in the end of the function and not do things like `if ... {return}`. For me, doing this in the very beginning of a function is an exception. IMHO this doesn't hurt readability like a `if ... {return}` in the middle of the function would do. When you start reading the function, it's very clear that it's a No-op in case `max_depth` is 0. Putting an else there would indent the rest of the function which I also don't like.